studio/__main_unit__.py

```diff
@@ -92,6 +92,11 @@ async def startup_event():
     logging.info('"Studio" application startup complete.')
 
 
+@app.get("/is_standalone", response_model=bool)
```

Comment:
@quanpython 
The category (Tag) of this API should be included in "others" (categories such as /aglolist).